aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -588,6 +588,7 @@ public static ModifyDbInstanceRequest modifyDbInstanceAfterCreateRequestV12(fina
 
     public static ModifyDbInstanceRequest modifyDbInstanceAfterCreateRequest(final ResourceModel model) {
         final ModifyDbInstanceRequest.Builder builder = ModifyDbInstanceRequest.builder()
+                .allowMajorVersionUpgrade(model.getAllowMajorVersionUpgrade())
```

Comment:
I am unsure if this change is intended to be included in this pull request, as it is not mentioned in the pull request description. Could you please confirm and update the pull request description? Alternatively, it may be better to create a separate pull request for this change. Combining two features in different resources within one pull request could lead to confusion and make it harder to track in the git commit history.